api/accounts/account.gen.go

```diff
@@ -57,10 +57,8 @@ func (n AccountReadResult) GetResponseMap() map[string]interface{} {
 	return n.response.Map
 }
 
-type (
```

Comment:
This appears to revert some gofumpt changes like this one.

Reply:
Ah I'll add running fmt at the end of api.